Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -532,8 +532,11 @@ def audit_conflicts
       end
     end
 
+    versioned_conflicts_whitelist = %w[node@ bash-completion@]
```

Comment:
I actually meant making it a true constant, since a whitelist seems to me like it should be a constant.